docs/ref/request-response.txt

```diff
@@ -290,6 +290,10 @@ Methods
 
     Example: ``"127.0.0.1:8000"``
 
+    Raises a ``django.core.exceptions.DisallowedHost`` exception if the host
+    is not in ``ALLOWED_HOSTS`` or the domain name is not valid according to
+    ``RFC 1034/1035``.
```

Comment:
I think `DisallowedHost` and `ALLOWED_HOSTS ` should be properly linked to their entry in the docs. You might as well also make the two RFC numbers external links, if you're including those in the text.

Reply:
@cjerdonek can I use the following links for RFC 1034 and 1035.
- https://tools.ietf.org/html/rfc1034
- https://tools.ietf.org/html/rfc1035

Reply:
Sure, I think so.